janusgraph-lucene/src/test/java/org/janusgraph/diskstorage/lucene/LuceneIndexTest.java

```diff
@@ -135,16 +128,12 @@ public void testSupport() {
         assertFalse(index.supports(of(Geoshape.class, Cardinality.SINGLE, new Parameter<>(MAPPING, Mapping.PREFIX_TREE)), Geo.DISJOINT));
     }
 
-//    @Override
-//    public void testDeleteDocumentThenModifyField() {
-//        // This fails under Lucene but works in ES
-//        log.info("Skipping " + getClass().getSimpleName() + "." + methodName.getMethodName());
-//    }
-
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testMapKey2Field_IllegalCharacter() {
-        index.mapKey2Field("here is an illegal character: " + REPLACEMENT_CHAR, null);
+        assertThrows(IllegalArgumentException.class, () ->{
+            index.mapKey2Field("here is an illegal character: " + REPLACEMENT_CHAR, null);
+        });
```

Comment:
I haven't actually tested this, but I think if we want to capture that the error message is as expected we can update this to:
```
Throwable exception = assertThrows(IllegalArgumentException.class, () ->{
    index.mapKey2Field("here is an illegal character: " + REPLACEMENT_CHAR, null);
});
assertEquals("here is an illegal character: " + REPLACEMENT_CHAR, exception.getMessage());
```

Reply:
Currently this test doesn't check the error message, so we shouldn't change the flow of the test in this PR. We can make other PRs for test improvements if needed.